### ISSUE TITLE
install should contain subdir

### ISSUE THREAD

**Issue by michauk** [Icinga/icingaweb2-module-reactbundle#3]
Hey,
I installed your module in order to have icinga/vsphere
Your tarball install instructions seems wrong :

&& mkdir "$MODULES_PATH" \

should be :

&& mkdir "$MODULES_PATH/reactbundle" \

or everything comes into /usr/share/icingaweb2/modules without subdir, thus preventing to enable the module, which directory doesn't exist + a bit messy in /usr/share/icingaweb2/modules after that ;)

same remark for git install?

regards

**Comment by michauk** [comment]
same for ipl and incubator?

**Comment by Thomas-Gelf** [comment]
You're right, thanks for reporting this. GIT instructions should be fine, wget/tar is wrong.

**Comment by michauk** [comment]
You're welcome.
By the way, at the end of the procedure, I had every module enabled but code errors for the final part (enabling vsphere-db if I remeber well).
I stumbled on a thread somewhere where you asked a guy to double-check the ipl/incubator release. I then saw I was running ipl/incubator release 0.0.0 (at least that was the info reported by the icingaweb2 admin page). So I grabbed the .tar directly and all went fine.
=> Maybe another problem in the installation directives? I did'nt investigate further.

(now I'm looking for instructions on how to use it all - I didn't find the vsphere/esxi checks but I didn'"t have much time to look further). It's strange that the vsphere doc tells everything to install and then.... nothing :)

regards
